### DISCUSSION TITLE
How nextjs.org loading font direct as chrome extension ?

### DISCUSSION THREAD

**Discussion by khanakia** [vercel/next.js#57940] [Help]
### Summary

When you visit https://nextjs.org/ it loading the fonts from here `chrome-extension://kbfnbcaeplbcioakkpcpgfkobkghlhen/src/fonts/8efb053b450b4a72/inter_Inter-Medium.woff`

How's that possible? If yes, then how i can achieve the same?

![Screenshot 2023-11-02 at 8 46 08 PM](https://github.com/vercel/next.js/assets/2174170/3d6a8b36-5b24-4ac4-b54a-161575a4fa7b)

### Example

https://nextjs.org/

**Comment by icyJoseph** [comment]
do you see the same if you use an incognito tab?

**Reply by khanakia** [reply]
No, incognito it does loads from here https://nextjs.org/_next/static/media/2ece3e04ee7a0792-s.p.woff2
